netket/operator/_potential.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional, Callable
+
+from netket.utils.types import DType
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import ContinousOperator
+
+import jax.numpy as jnp
+
+
+class PotentialEnergy(ContinousOperator):
+    def __init__(
+        self, hilbert: AbstractHilbert, afun: Callable, dtype: Optional[DType] = float
+    ):
+        r"""Args:
+        afun: The potential energy as function of x_in
+        """
+
+        self._afun = afun
+        self._dtype = dtype
+        super().__init__(hilbert)
+
+    @property
+    def dtype(self) -> DType:
+        return self._dtype
+
+    def expect_kernel(self, logpsi, params, x_in, data):
+        r"""
```

Comment:
can we just rename x_in to x? can you also add docstrings for all the arguments?